test-suite/shared/src/test/scala/org/scalajs/testsuite/compiler/BooleanTest.scala

```diff
@@ -16,20 +16,81 @@ import org.junit.Test
 import org.junit.Assert._
 
 class BooleanTest {
-  @Test def bitwiseAndOrXorOperators(): Unit = {
-    assertFalse(false & false)
-    assertFalse(false & true)
-    assertFalse(true & false)
-    assertTrue(true & true)
-
-    assertFalse(false | false)
-    assertTrue(true | false)
-    assertTrue(false | true)
-    assertTrue(true | true)
-
-    assertFalse(false ^ false)
-    assertTrue(true ^ false)
-    assertTrue(false ^ true)
-    assertFalse(true ^ true)
+  @noinline def hideFromOptimizer(x: Boolean): Boolean = x
+
+  // See #220, #409
```

Comment:
How did you find these :rofl: the comment thread is a blast from the past

Reply:
I followed the git history through 3 or 4 renames and 1 rewrite. 😅